tier/plural/recipes/tier-azure.yaml

```diff
@@ -4,10 +4,15 @@ provider: AZURE
 dependencies:
   - repo: bootstrap
     name: azure-k8s
+  - repo: ingress-nginx
+    name: ingress-nginx-azure
 
 sections:
   - name: tier
-    configuration: []
+    configuration:
```

Comment:
so you'd probably need to add the helm values configuration for this in `tier/helm/tier/values.yaml.tpl` to wire up the ingress appropriately.  I think `strapi` is a pretty good model of this which just uses the helm starter templates to configure a simple stateless app w/ an ingress on top

Reply:
Pushed a commit to address this.